src/bytes.ts

```diff
@@ -10,16 +11,16 @@ export const zeros = function(bytes: number): Buffer {
 }
 
 /**
- * Left Pads an `Array` or `Buffer` with leading zeros till it has `length` bytes.
+ * Left Pads a `Buffer` with leading zeros till it has `length` bytes.
  * Or it truncates the beginning if it exceeds.
- * @param msg the value to pad (Buffer|Array)
+ * @param msg the value to pad (Buffer)
  * @param length the number of bytes the output should be
  * @param right whether to start padding form the left or right
- * @return (Buffer|Array)
+ * @return (Buffer)
  */
-export const setLengthLeft = function(msg: any, length: number, right: boolean = false) {
+export const setLengthLeft = function(msg: Buffer, length: number, right: boolean = false) {
```

Comment:
Hmm, this is pretty confusing to have this boolean `right` parameter in the API. Can we instead do an internal `setLength` method and move the logic over there and do function signature (so: `function(msg: Buffer, length: number)`) and internal structure analogue to `setLengthRight`?

Reply:
Yes, that's definitely nicer.

Reply:
Addressed in a2a50ae